src/convertToObject.js

```diff
@@ -6,7 +6,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (sourceString.length === 0) {
+    return {};
+  }
+
+  const GROUP_STYLE = sourceString.trim().split(';');
+  const GET_CLEAN_STYLE = GROUP_STYLE.map((pair) => pair.trim()).filter(
+    (style) => style !== '',
+  );
+
+  const SEPARATION_STYLE = GET_CLEAN_STYLE.map((pair) => {
+    return pair.trim().split(':');
+  });
+  const GET_STYLE_IN_OBJECT = SEPARATION_STYLE.reduce((acc, style) => {
+    acc[style[0].trim()] = style[1].trim();
+
+    return acc;
+  }, {});
```

Comment:
The variable name 'GET_STYLE_IN_OBJECT' does not follow naming conventions and is not descriptive. Consider renaming it to 'stylesObject' or similar. Also, the reduce function's accumulator parameter could be named more descriptively, such as 'styles' instead of 'acc'.